linode/manager
# refactor: [M3-6309] - SRC > Components > Currency

## Description 📝

- Cleans up the `<Currency />` component
- Also cleans up some components that consume the `<Currency />` component

## How to test 🧪

- Make sure prices show up as expected in Cloud Manager
- Test that the changes to `InvoiceDetail.tsx` did not break functionality (this was just a switch from `withRouter` to `useParams` hook)
